lib/extension_file_generator.php

```diff
@@ -97,7 +97,7 @@ public function parseAndOutput()
         // Set extension name variables
         $data['name'] = $this->options['name'];
         $data['snake_case_name'] = str_replace(' ', '_', strtolower($data['name']));
-        $data['class_name'] = str_replace(' ', '', ucwords(str_replace('_', ' ', $data['name'])));
+        $data['class_name'] = str_replace(' ', '', ucwords(str_replace('_', ' ', strtolower($data['name']))));
```

Comment:
Wouldn't it be better to use `$data['class_name'] = Loader::toCamelCase($data['snake_case_name']);`? Loader::toCamelCase() does pretty much what is needed here.

Reply:
Updated